### DISCUSSION TITLE
content security policy

### DISCUSSION THREAD

**Discussion by cis-lokesh** [mozilla/hubs#4433] [General Questions & Discussion] [answered]
Hi 
I've tried adding my library cdn url  to he server setting content security page  from admin panel but still I'm unable to use tto use the library instead I get this.

Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'sha256-MIOmbi3/142y0xbs7/3VMZz8xGn6epGa6/PPJfNse/c=' 'sha256-1KTw8KSnt4ThStf9gMsRwmxbeNA0APO8rjwwO0f51No=' https://cdn.jsdelivr.net https://hubs-mynew-world-proxy.lokesh5707.workers.dev  'self' 'unsafe-eval' 'sha256-ViVvpb0oYlPAp7R8ZLxlNI6rsf7E7oz8l1SgCIXgMvM=' 'sha256-hsbRcgUBASABDq7qVGVTpbnWq/ns7B+ToTctZFJXYi8=' 'sha256-MIpWPgYj31kCgSUFc0UwHGQrV87W6N5ozotqfxxQG0w=' 'sha256-buF6N8Z4p2PuaaeRUjm7mxBpPNf4XlCT9Fep83YabbM=' 'sha256-/S6PM16MxkmUT7zJN2lkEKFgvXR7yL4Z8PCrRrFu4Q8=' https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.js 'sha256-foB3G7vO68Ot8wctsG3OKBQ84ADKVinlnTg9/s93Ycs=' 'sha256-g0j42v3Wo/ohUAMR/t0EuObDSEkx1rZ3lv45fUaNmYs=' https://www.google-analytics.com https://ssl.google-analytics.com  https://hubs-mynew-world-proxy.lokesh5707.workers.dev  https://aframe.io https://www.youtube.com https://s.ytimg.com". Either the 'unsafe-inline' keyword, a hash ('sha256-hVJyl4+0ULm8mhXVlovSL2WsEc/5oHPv8rYWADLbOxs='), or a nonce ('nonce-...') is required to enable inline execution.

**Comment by shamsail** [comment, marked as answer]
try this to include in script-src with other scripts urls with space between sources from admin panel 
```
'sha256-hVJyl4+0ULm8mhXVlovSL2WsEc/5oHPv8rYWADLbOxs='
```

**Comment by pattersonbl2** [comment]
Marking as answered for now, please unmark if help is still needed.
